ReportUtilityBills/measure.rb

```diff
@@ -231,37 +257,54 @@ def run(runner, user_arguments)
       get_utility_bills(fuels, utility_rates, utility_bills, utility_bill_scenario, @hpxml.header)
 
       # Write/report results
-      report_runperiod_output_results(runner, utility_bills, output_format, output_path, utility_bill_scenario.name)
+      report_runperiod_output_results(runner, args, utility_bills, output_path, utility_bill_scenario.name, @hpxml.header)
     end
 
     return true
   end
 
-  def report_runperiod_output_results(runner, utility_bills, output_format, output_path, bill_scenario_name)
-    segment = utility_bills.keys[0].split(':', 2)[0]
-    segment = segment.strip
+  def report_runperiod_output_results(runner, args, utility_bills, output_path, bill_scenario_name, _header)
```

Comment:
Was `header` supposed to be used? If not, remove it.

Reply:
Hmm... maybe I intended to get run period from it at one point? Not sure. I will remove.